### PR TITLE
Version 1.4.0

### PR DESCRIPTION
**Pull request by Sandertv**
This pull request is for the Version 1.4.0 update of BlockSniper.

Most planned features have been finished, but insufficiently tested.
Therefore, don't merge this yet.

If you want to test this, please do so. More information can be found at #51

### CODE REVIEW

File: src/BlockHorizons/BlockSniper/undo/Undo.php [hunk 1]
```diff
@@ -27,11 +27,18 @@ public function restore() {
 	public function getDetachedRedo(): Redo {
 		$redoBlocks = [];
 		foreach($this->undoBlocks as $undoBlock) {
-			$redoBlocks[] = $undoBlock->getLevel()->getBlock($undoBlock);
```
Comment (Sandertv) [inline src/BlockHorizons/BlockSniper/undo/Undo.php]: I have been sleeping when making this. No idea how this ended up here. 😋

**Comment by TheDiamondYT1** [comment]
I've been updating the schematics repo... It can now be used as a virion and has some method changes and additions. This will need to be incorporated in here

**Comment by Sandertv** [comment]
Alright, thanks!

**Comment by TheDiamondYT1** [comment]
So you just make this PR a 1.5.0 PR and contradicted your comment in the 1.4 issue 😂.

**Comment by Sandertv** [comment]
I messed everything up again. 😂 Sorry bro, I prove again I'm a git noob.

**Comment by TheDiamondYT1** [comment]
I'll fix it

**Comment by TheDiamondYT1** [comment]
Removed the commit @Sandertv

**Comment by Sandertv** [comment]
👍

**Pull request merged**